src/main/java/core/basesyntax/controller/ControllerHandler.java

```diff
@@ -0,0 +1,85 @@
+package core.basesyntax.controller;
+
+import core.basesyntax.dao.BetDao;
+import core.basesyntax.dao.BetDaoImpl;
+import core.basesyntax.dao.UserDao;
+import core.basesyntax.dao.UserDaoImpl;
+import core.basesyntax.model.Bet;
+import core.basesyntax.model.User;
+import java.util.Scanner;
+
+public class ControllerHandler {
+    public static final String INVALID_INPUT_ARGS_MESSAGE =
+            "Please, make sure your input data meets requirements";
+    public static String INPUT = new Scanner(System.in).nextLine();
```

Comment:
let's not store such values as static fields
create only one scanner inside your method and call `nextLine` when needed